R/validate_inputs.R

```diff
@@ -1,5 +1,5 @@
 
-#' send input validation messages to output
+#' Send input validation messages to output
```

Comment:
Do we have a convention for formats of help files fields?

Reply:
be consistent with the others.